.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+	"plugins": [
+		"jasmine",
+        "mocha"
+	],
+    "extends": "standard",
+    "rules": {
+        "semi": [2, "always"],
+        "indent": 0,
+        "mocha/no-exclusive-tests": "error",
+        "space-before-function-paren": 0,
+
+        /* These should be removed */
+        "no-throw-literal" : 0,
```

Comment:
These rules here are additional exclusions to get the linter to run successfully even though they are valid complaints. Someone should remove this section and **fix** the relevant errors.